news/2023/2023-02-10-osumonthly-7.md

```diff
@@ -0,0 +1,263 @@
+
+## Skinning Recap
+
+### Skinning news
+
+#### The osu! 2023 Skinning Poll
+
+On January 16th an official poll has been opened up to the public in order to gauge their opinion on the state of skinning, both from the skinning community's perspective, but also from the entire player base. By the time this edition of osu!monthly is released the poll will have been closed, however you can still contribute [through the forum](https://osu.ppy.sh/community/forums/15) if you think you have an opinion worth sharing.
+
+#### Skin of the Year 2023 voting is now live
+
+The skinning community has been hard at work creating new skins for others to enjoy, so we highly encourage you to check out the [news post](soon) and cast your votes!
+
+### Featured skins of the month
+
+As should be tradition by now, 3 skins from the month of January have been selected to be showcased on osu!monthly:
+
+- [Skadi Arknights](https://osu.ppy.sh/community/forums/topics/1706882?n=1) - Inspired by Skadi from *Arknights*, this skin revolves around an obvious theme while still keeping to minimalism, especially in the gameplay section
+- [Pink Bean - MapleStory](https://osu.ppy.sh/community/forums/topics/1700282?n=1) - Pink Bean from *Maplestory* is a memorable character, and this skin pays homage to both the original game and the beloved character with its style and design choices
+- [5WC 2023](https://osu.ppy.sh/community/forums/topics/1699533?n=1) - A skin made specially for the upcoming 5 Digit World Cup 2023 Tournament. Created in the author's consistent and gorgeous design style, as well as following this year's branding for 5WC, it truly shows how much love and care the organizers put into their tourneys
+
+<iframe width="100%" height="315" src="https://www.youtube.com/embed/videoseries?list=PLA2Nwtxa4WMorBTgwYc7puyftc30_h1Ta" frameborder="0" allowfullscreen></iframe>
+
+## Did you know?
+
+The art and name for osu!stream came from a community member! In mid 2010, [Koko Ban](https://osu.ppy.sh/users/60603) on the forums would [show off their idea](https://osu.ppy.sh/community/forums/topics/13087?n=645) for another osu! game by posting a fan concept. The community received it really well, and when peppy decided to work on the iOS version of the game, Koko Ban was commissioned to do the full visual game design. Thanks once again to [Kurboh](https://osu.ppy.sh/users/15243233) for providing this cool titbit of osu! lore.
+
+Source: ![](/wiki/shared/news/2023-02-10-osumonthly-7/did-you-know.jpg)
+
+## Monthly community quiz
+
+We admit that the last month's question was brutal, and we'll be looking into balancing the difficulty over the next few issues.
+The correct answer was '05Z2 Lazer Radiation', referring to the presence of a 'Perfect Curve' slider type in the osu!lazer editor, along with this banner that was at the bottom of some (very) early osu!(lazer) builds.
```

Comment:
hmm, well "Class 05Z2 Lazer Radiation" is not a "class" or "classification" (it's "lazer" radiation), while the "05Z2" classification would be

in fact just "05Z2" makes way more sense and would be in line with what the question asks about @chromb 

```suggestion
We admit that the last month's question (*What class must mappers experience to create perfect sliders?*) was brutal, and we'll be looking into balancing the difficulty over the next few issues.

The correct answer was "05Z2", referring to the presence of a "perfect curve" slider type in the osu!(lazer) editor, along with this banner that was at the bottom of some (very) early builds.
```

Reply:
Yeah that works better 👍

Reply:
dunno why this didn't auto close but I applied the suggestion